src/config/media-viewer-config.interface.ts

```diff
@@ -0,0 +1,7 @@
+import { Config } from './config.interface';
+
+export interface MediaViewerConfig extends Config {
+  enable: boolean;
```

Comment:
Is there a reason to have a separate `enable` boolean? wouldn't simply setting the `image` or `video` boolean to true imply that the viewer is enabled?

Reply:
Yes its a bit redundant so i fixed it